app/renderer/sweetalert.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+//Methods interface with jQuery
+/* jshint ignore:start */
```

Comment:
Do we need a full jshint ignore? we can set /* globals swal */ afaik

Reply:
dont do a full review yet. Yes of course we can set it.

Reply:
Oh, sorry, I'm just pointing out the things I saw